multimodal/tests/unittests/test_predictor.py

```diff
@@ -59,6 +59,22 @@ def verify_predictor_save_load(predictor, df, verify_embedding=True, cls=MultiMo
             assert embeddings.shape[0] == len(df)
 
 
+def verify_realtime_inference(predictor, df, verify_embedding=True):
```

Comment:
Just curious, will there be a case when we don't want to verify embedding?

Reply:
It seems that extracting embedding may be used in all kinds of pipelines and models.